schema/sale/index.js

```diff
@@ -168,6 +168,12 @@ const SaleType = new GraphQLObjectType({
         type: GraphQLBoolean,
         resolve: ({ auction_state }) => auction_state === "preview",
       },
+      is_registration_closed: {
+        type: GraphQLBoolean,
+        resolve: ({ auction_state, registration_ends_at }) => {
+          return auction_state === "closed" || moment().isAfter(registration_ends_at)
```

Comment:
I'm not totally sure we need the `auction_state == 'closed'` here, as we risk saturating this field (which otherwise corresponds to a single field returned from the API). Clients should be dealing with auction closing separately from registration ending.